### ISSUE TITLE
server and autoexec.cfg are just replaced with empty files

### ISSUE THREAD

**Issue by fipso** [kaimallea/csgo#50]
Probably broke in the last commit

EDIT:
so if i delete my server.cfg a new one with the default settings is created, but the autoexec file is never created.

my start params:

```bash
#!/bin/bash

PLUGINS=$(cat <<-END
https://mms.alliedmods.net/mmsdrop/1.10/mmsource-1.10.7-git971-linux.tar.gz
https://sm.alliedmods.net/smdrop/1.10/sourcemod-1.10.0-git6478-linux.tar.gz
http://users.alliedmods.net/~kyles/builds/SteamWorks/SteamWorks-git131-linux.tar.gz
https://bitbucket.org/GoD_Tony/updater/downloads/updater.smx
https://github.com/splewis/csgo-practice-mode/releases/download/1.3.3/practicemode_1.3.3.zip
https://github.com/splewis/csgo-pug-setup/releases/download/2.0.5/pugsetup_2.0.5.zip
https://github.com/splewis/csgo-retakes/releases/download/v0.3.4/retakes_0.3.4.zip
https://github.com/b3none/retakes-instadefuse/releases/download/1.4.0/retakes-instadefuse.smx
https://github.com/b3none/retakes-autoplant/releases/download/2.3.0/retakes_autoplant.smx
https://github.com/b3none/retakes-hud/releases/download/2.2.5/retakes-hud.smx
https://ptah.zizt.ru/files/PTaH-V1.1.3-build20-linux.zip
https://github.com/kgns/weapons/releases/download/v1.7.1/weapons-v1.7.1.zip
https://github.com/kgns/gloves/releases/download/v1.0.5/gloves-v1.0.5.zip
END
)

# https://github.com/shanapu/MyWeaponAllocator
# https://github.com/Franc1sco/FixHintColorMessages

docker run \
  --rm \
  --interactive \
  --tty \
  --detach \
  --mount type=bind,source=$(pwd),target=/home/steam/csgo \
  --network=host \
  --env "SERVER_HOSTNAME=BoeckelsEsport" \
  --env "SERVER_PASSWORD=mbeezy" \
  --env "STEAM_ACCOUNT=XXX" \
  --env "AUTHKEY=XXX" \
  --env "SOURCEMOD_ADMINS=XXX" \
  --env "INSTALL_PLUGINS=${PLUGINS}" \
  --env "RETAKES=1" \
  kmallea/csgo
```

**Comment by Soren90** [comment]
Hey,

Sorry for the inconvenience. I've added a new PR to fix the issue. #51 

What you can do while waiting for @kaimallea approval, you need to clone the repo and build the image yourself.

**Comment by fipso** [comment]
Thanks a lot @Soren90

**Comment by kaimallea** [comment]
fixed by #51
